src/services/copyConfig/consent/consentConfiguration.test.js

```diff
@@ -44,7 +51,7 @@ describe('ConsentConfiguration test suite', () => {
       .mockResolvedValueOnce({ data: getResponseWithContext(expectedGigyaResponseOk, 'consent_legalStatement_terms.consentId2_pt', apiKey) })
       .mockResolvedValueOnce({ data: getResponseWithContext(expectedGigyaResponseOk, 'consent_legalStatement_terms.termsConsentId1_en', apiKey) })
 
-    const responses = await consentConfiguration.copy(apiKey, { dataCenter }, consentOptions.getOptions())
+    const responses = await consentConfiguration.copy(apiKey, { dataCenter }, optionsConsent)
```

Comment:
Use the class ConsentOptions instead of an arbitrary object.
The branches array should be in ConsentOptions class